server/src/libs/environment/env.ts

```diff
@@ -5,13 +5,15 @@ import { config as dotEnvConfig } from 'dotenv'
 
 import { parseBooleanVariables } from './parseBooleanVariables'
 
-const pathDevSource = join(__dirname, '../../..', `.env.${process.env.ENV}`)
+const pathDevSource = join(__dirname, '../../..', `.env.${process.env.ENV || ''}`)
```

Comment:
je propose de casser en amont avec quelque chose du genre

```suggestion
if (typeof process.env.ENV === 'undefined') {
	throw new Error('The "ENV" environnement variable must be set')
}

const pathDevSource = join(__dirname, '../../..', `.env.${process.env.ENV}`)
```